### PR TITLE
Add subscriptions ui fields

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Add subscriptions ui fields

Update links on list to not allow editing Cancelled
Rename subscriptionFactory variable

[stage-19]

## Motivation and Context
Billing Frictions epic

## How Has This Been Tested?
Tested changes locally. Will update unit tests separately.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
More UI updates and some fixes.

@ezequielc @santiagonoguez please review. Thanks!

File: web/partials/billing/subscription.html [hunk 1]
```diff
@@ -10,13 +10,104 @@ <h1 class="app-header-title">Subscription</h1>
...
+          <a id="removeLicenses" class="btn btn-default btn-block mt-3" ui-sref="apps.billing.licenses.remove({subscriptionId: subscriptionFactory.item.subscription.id})">Remove Display Licenses</a>
+          <a id="manageLicenses" class="btn btn-default btn-block mt-3" ui-sref="apps.billing.licenses.manage({subscriptionId: subscriptionFactory.item.subscription.id})" ng-if="false">Manage Network Licenses</a>
+        </div>
+      </div>
+    </div>
+    <div class="col-md-4">
+      <div class="border-container subscription-card">
+        <div class="panel-body">
+          <label ng-show="subscriptionFactory.item.subscription.auto_collection === 'off'">Next Invoice</label>
+          <label ng-show="subscriptionFactory.item.subscription.auto_collection !== 'off'">Next Payment</label>
```
Comment (santiagonoguez) [inline web/partials/billing/subscription.html:36]: so auto_collection means whether it's automatically charged ( to credit card ), or invoiced ? I hadn't realized that :O

**Review by santiagonoguez — APPROVED** [review]
@alex-deaconu code looks good to me, and page update looks good also, thanks !

File: web/partials/billing/subscription.html (hunk 1 above)
Comment (alex-deaconu) [reply web/partials/billing/subscription.html:36]: @santiagonoguez yes, there are two values. One on the Customer and one on the Subscription. The Customer one is the default but could be overridden for a particular Subscription. Thanks.

**Pull request merged**